coverage

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -Cefu -o pipefail
+
+root="$(bazel info workspace)"
+cd "${root:?}" || exit
+
+file="$(mktemp)" || exit
+trap 'rm -f -- "${file}"' EXIT
+
+bazel coverage -- //... | tee -- "${file:?}" || exit
+echo
+
+sed -rn -e 's|^  (/.+/coverage\.dat)$|\1|p' -- "${file:?}" \
```

Comment:
If I read correctly, the sed is just removing the leading spaces, but they shouldn't be significant with the call to xargs. If so, it could be replaced with a grep?

Reply:
yeah, but I guess it's preferrable to also use `-d '\n'` here, which is more robust, but breaks this again.
I've now switched to `mapfile`, which seems a bit more robust in that it guarantees that we actually pass all arguments in a single invocation.